tests/test_directive_code.py

```diff
@@ -356,6 +356,8 @@ def test_code_block_emphasize_latex(app, status, warning):
     latex = (app.outdir / 'Python.tex').text(encoding='utf-8').replace('\r\n', '\n')
     includes = '\\fvset{hllines={, 5, 6, 13, 14, 15, 24, 25, 26, 27,}}%\n'
     assert includes in latex
+    includes = '\\end{sphinxVerbatim}\n\sphinxresetverbatimhllines\n'
```

Comment:
invalid escape sequence fixed at d210da9. Thanks @jdufresne, sorry for my oversight. I will backport fix to 1.8 branch.